### ISSUE TITLE
[BUG] Native HTML background is showing when keyboard appears.

### ISSUE THREAD

**Issue by Lyokone** [the-airbender/quill_html_editor#19]
On iOS, a white background is shown behind the editor when the keyboard opens.
See video

https://user-images.githubusercontent.com/3680002/218483561-55ef3b89-7e08-40d6-8b75-f1b1ad52cd77.MP4

Reproduction code:
```
    return Scaffold(
      backgroundColor: Colors.red,
      appBar: AppBar(
        title: const Text('Compose'),
      ),
      body: Stack(
        children: [
          ListView(
            children: [
              QuillHtmlEditor(
                hintText: 'New message',
                controller: controller,
                height: 400,
                defaultFontSize: 14,
                isEnabled: true,
                backgroundColor: Colors.red,
              ),
            ],
          ),
        ],
      ),
    );

```

**Comment by the-airbender** [comment]
Hi @Lyokone,
Thanks for the sample code. Could you please confirm if this is being tested on a simulator or a physical device?

**Comment by the-airbender** [comment]
> 

Thanks! Will check it out.

**Comment by the-airbender** [comment]
Hi @Lyokone ,
This bug has been resolved and deployed with the new version [v2.0.7](https://pub.dev/packages/quill_html_editor)
Thanks!

**Comment by Lyokone** [comment]
Thanks for the quick fix!
